docs/concepts/structured_views.md

```diff
@@ -0,0 +1,39 @@
+# Concept: Structured Views
+
+Structured views are a type of [view](../concepts/views.md), which provide a way for developers using db-ally to define what they need from the LLM, including:
+
+* The desired data structure, such as the specific fields to include from the data source.
+* A set of operations the LLM may employ in response to natural language queries (currently only “filters” are supported, with more to come)
```

Comment:
I'd love to see alink to filters but we do not have any page, so maybe we should create one? Maybe a header in the IQL concept page?